.github/workflows/test.yml

```diff
@@ -16,8 +16,11 @@ jobs:
       - name: Build
         run: yarn build
 
-      - name: Start MongoDB for unit tests
-        run: docker run -d -p 27017:27017 -v ~/data:/data/db mongo
+      - name: Start required docker containers for unit tests
```

Comment:
should we use services in github actions?
https://docs.github.com/en/actions/guides/about-service-containers